### ISSUE TITLE
Recommendation for decoder finetuning

### ISSUE THREAD

**Issue by elias-ramzi** [FoundationVision/LlamaGen#61]
Hi,

I am trying to finetune the decoder only of the tokenizer on a new dataset. I was wondering if you could share some finetuning recipies.

Mainly:

- Should the discriminator loss be used / finetuned at the same time on the new dataset
- Should I activate the discriminator loss directly at the begining of training
- Should I keep the same hyperparameters as training (lr=1e-4, weight_decay=0.05)
- Should I use the optimizer state of the pretraining

Thank you for your help!
